pkg/apis/build/v1alpha1/buildrun_types.go

```diff
@@ -45,8 +45,21 @@ type BuildRunSpec struct {
 	// image would be pushed to. It will overwrite the output image in build spec
 	// +optional
 	Output *Image `json:"output,omitempty"`
+
+	// State is used for canceling a buildrun (and maybe more later on).
```

Comment:
```suggestion
	// State is used for canceling a buildrun.
```

Reply:
I pulled this directly from the analogous tekton godoc ... I'll ponder while I figure out what happened with my ubuntu make generate-crds output

Reply:
yeah I'm going to leave this one as is